packages/@webex/plugin-meetings/src/breakouts/utils.ts

```diff
@@ -13,3 +17,22 @@ export const getBroadcastRoles = (options): string[] => {
 
   return recipientRoles;
 };
+
+/**
+ * Deals with all kinds of errors of breakout service
+ * @param {object} error // one of the breakout service error
+ * @returns {object}
+ */
+export const boServiceErrorHandler = (error: any): any => {
+  const errorCode = error?.body?.errorCode;
+  const {EDIT_LOCK_TOKEN_MISMATCH} = BREAKOUTS.ERROR_CODE;
+  switch (errorCode) {
+    case EDIT_LOCK_TOKEN_MISMATCH:
+      LoggerProxy.logger.info(`Breakouts#clearSessions --> Edit lock token mismatch`);
```

Comment:
issue: logger info message now misleading as it says "clearSessions" but it is called from multiple places. Perhaps you could pass in a logging string to `boServiceErrorHandler `

Reply:
yes, need to update it.